specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json

```diff
@@ -1450,20 +2063,43 @@
       "required": true,
       "type": "string"
     },
+    "resourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "description": "Azure Resource Group Name.",
+      "required": true,
+      "type": "string"
+    },
     "budgetNameParameter": {
-      "name": "name",
+      "name": "budgetName",
       "in": "path",
-      "description": "Budget name.",
+      "description": "Budget Name.",
       "required": true,
-      "type": "string",
-      "x-ms-parameter-location": "method"
```

Comment:
@bgsky why was `"x-ms-parameter-location": "method"` removed? This causes a breaking change in the client. This makes budgetName a property on the client from being a method parameter.

Reply:
@amarzavery Refer to https://github.com/Azure/azure-rest-api-specs/pull/2386 for a fix.